UncaughtPromiseError > OwlError

Uncaught Promise > An error occured in the owl lifecycle (see this Error's "cause" property)

Occured on localhost:18069 on 2025-04-25 03:55:00 GMT

OwlError: An error occured in the owl lifecycle (see this Error's "cause" property)
    Error: An error occured in the owl lifecycle (see this Error's "cause" property)
        at handleError (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:961:101)
        at App.handleError (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:1608:29)
        at Fiber._render (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:986:19)
        at Fiber.render (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:984:6)
        at ComponentNode.initiateRender (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:1054:47)

Caused by: EvalError: Can not evaluate python expression: (bool(account_type not in ['asset_receivable', 'liability_payable']))
    Error: Name 'account_type' is not defined
    EvalError: Can not evaluate python expression: (bool(account_type not in ['asset_receivable', 'liability_payable']))
    Error: Name 'account_type' is not defined
        at evaluateExpr (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:3337:54)
        at evaluateBooleanExpr (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:3340:8)
        at AccountMoveLineListRenderer.evalColumnInvisible (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:9390:45)
        at http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:9335:147
        at Array.filter (<anonymous>)
        at get optionalFieldGroups (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:9335:113)
        at AccountMoveLineListRenderer.slot1 (eval at compile (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:1563:421), <anonymous>:46:73)
        at callSlot (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:1155:25)
        at DropdownPopover.template (eval at compile (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:1563:421), <anonymous>:28:10)
        at node.renderFn (http://localhost:18069/web/assets/bacb846/web.assets_web.min.js:1104:207)

--- a/account_payment_partner/views/account_move_line.xml
+++ b/account_payment_partner/views/account_move_line.xml
@@ -31,7 +31,7 @@
                     name="payment_mode_id"
                     optional="hide"
                     force_save="1"
-                    column_invisible="account_type not in ['asset_receivable', 'liability_payable']"
+                    invisible="account_type not in ['asset_receivable', 'liability_payable']"
                     readonly="account_type not in ['asset_receivable', 'liability_payable'] or reconciled"
                 />
             </field>

